aaad/aaad_app.py

```diff
@@ -0,0 +1,159 @@
+from flask import Flask, jsonify, request, render_template, redirect, make_response, flash, Response
+from flask_restful import abort, Api, Resource
+from flask_login import login_user, logout_user, login_required, current_user
+from webargs import fields
+from webargs.flaskparser import use_args
+import os
+import json
+import logging
+
+from authenticators import FileAuthenticator
+from authorizers import FileAuthorizer
+from resources import Users, Groups, CanActAsUsers
+from sessions import login_manager, SessionUser
+import sessions
+
```

Comment:
There is a redundant import here. Also, I believe it's best practice to prepend each imported module with a dot character when doing local imports. 

Reference: https://docs.python.org/2.5/whatsnew/pep-328.html